java-frontend/src/main/java/org/sonar/java/resolve/Resolve.java

```diff
@@ -512,15 +615,11 @@ private JavaSymbol selectBest(Env env, JavaType site, List<JavaType> argTypes, J
     return formalParameters;
   }
 
-  private static boolean usesTypeParameter(JavaSymbol.MethodJavaSymbol methodSymbol) {
-    return !methodSymbol.typeVariableTypes.isEmpty();
-  }
-
   /**
    * @param argTypes types of arguments
    * @param formals  types of formal parameters of method
    */
-  private boolean isArgumentsAcceptable(List<JavaType> argTypes, List<JavaType> formals, boolean isVarArgs, boolean autoboxing, boolean isParameterized) {
+  private boolean isArgumentsAcceptable(List<JavaType> argTypes, List<JavaType> formals, boolean isVarArgs, boolean autoboxing) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "isArgumentsAcceptable" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)